src/css/styles.css

```diff
@@ -129,6 +129,31 @@ body {
     background-color: #EEE;
 }
 
+.table {
```

Comment:
All these rules should definitely not be applied to `.table`—that’s a Bootstrap class that is meant to add generic table styling. Since these styles are not really related Bootstrap are also also *definitely* not generic to all tables, we should probably be targeting this table more specifically with a selector like:

    .container-list-view table

Or by adding a special class to the table in question, e.g. `container-list-view--list`.

Reply:
Yes, you're totally right! I want to go with the BEM naming using `container-list-view--list`. I haven't had much time to read up on all the intricacies but one thing I gleaned was that everything should be given a class and inheritance not used as much. In that case, should we use a class for all the `tr` elements as well and how would that name look like?

Reply:
Oh, I think it’s really easy to take this too far. You know that `.container-list-view--list` is *always* necessarily going to be a table, so targeting the `tr` elements in it with a special class feels unnecessary to me (same with, say, a class on an `ol/ul` and its `li` children).

On the other hand, it *would* be a good idea to use a special class for `th/td` elements in the table that should be treated specially (e.g. a differently aligned or special-width column) rather than using `nth-child()` selectors because it makes adding and removing columns and refactoring much easier.